{{cookiecutter.project_slug}}/compose/production/django/Dockerfile

```diff
@@ -1,5 +1,5 @@
 {% if cookiecutter.js_task_runner == 'Gulp' -%}
-FROM node:8-alpine as client-builder
+FROM node:8-jessie-slim as client-builder
```

Comment:
Shouldn't this be `node:10-stretch-slim`?

- [`stretch` is latest stable Debian release](https://www.debian.org/releases/)
- [`10` is the current Node LTS](https://nodejs.org/en/download/releases/)

Reply:
Yes, good point. I haven't tested yet but it should be easy to do on the generated project.